tee-worker/litentry/pallets/identity-management/src/lib.rs

```diff
@@ -204,14 +203,15 @@ pub mod pallet {
 
 	impl<T: Config> Pallet<T> {
 		// build the prime identity which is always a substrate address32-based identity
-		fn build_prime_identity(who: &T::AccountId) -> Result<Identity, DispatchError> {
-			let address_raw: [u8; 32] =
-				who.encode().try_into().map_err(|_| DispatchError::Other("Invalid AccountId"))?;
-			Ok(Identity::Substrate(address_raw.into()))
+		fn build_prime_identity(address: &LitentryMultiAddress) -> Result<Identity, DispatchError> {
```

Comment:
We don't need `Result` here

Reply:
After change `LitentryMultiaddress` -> `Identity`  `Result` is required.